docs/en_US/Tuner/InstallCustomizedTuner.rst

```diff
@@ -32,19 +28,26 @@ This command build a whl file which is a pip installation source.
 
 Step 2: Run command:
 
-``nnictl package install dist/demo_tuner-0.1-py3-none-any.whl``
+``pip install dist/demo_tuner-0.1-py3-none-any.whl``
+
+Register the customized tuner as builtin tuner:
+-----------------------------------------------
+
+Run following command:
+
+``nnictl algo register --meta meta_file.yml``
```

Comment:
how to write `meta_file.yml`?

Reply:
In fact we have a doc called `How to register customized algorithms as builtin tuners, assessors and advisors`, and it has more details, I don't know if this doc is necessary.

Reply:
what is the path of this doc?

Reply:
docs/en_US/Tutorial/InstallCustomizedAlgos.rst
https://nni.readthedocs.io/en/v2.0/Tutorial/InstallCustomizedAlgos.html

Reply:
yeah, it is duplicated, suggest to merge content of the two files together